app/controllers/admin/user_sign_transfers_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Admin
+  class UserSignTransfersController < Admin::ApplicationController
+    def new
+      redirect_to root_path, alert: t(".old_owner_required") if old_owner.nil?
+    end
+
+    def create
+      SignOwnershipTransferService.new.transfer_sign(old_owner: old_owner, new_owner: new_owner)
+
+      redirect_to admin_user_path(old_owner), notice: t(".success")
+    end
+
+    private
+
+    def user_sign_transfer_params
+      params.permit(:old_owner, :new_owner)
+    end
+
+    def old_owner
+      @old_owner ||= User.find_by(id: user_sign_transfer_params[:old_owner])
```

Comment:
You should use `find` so that an exception is raised if the user cannot be found (and we expect them to be there)
You might also consider requiring the param directly since you have quite a simple param structure:

```suggestion
      @old_owner ||= User.find(params[:old_owner])
```

Same advice goes for new owner below

Reply:
Also noting you're using different rules in the view that you should consider using here as well - specifically that the new owner shouldn't be the system user or the current owner.